app/assets/stylesheets/components/_form.scss

```diff
@@ -16,3 +16,13 @@
   margin-left: .5rem;
   vertical-align: baseline;
 }
+
+input[type='number'] {
+  -moz-appearance: textfield;
+}
+
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+}
```

Comment:
Ah, i see.

I think you want you add a `margin: 0` property to the second css snippet too (for older Chrome browsers, per http://stackoverflow.com/questions/3790935/can-i-hide-the-html5-number-input-s-spin-box)

Do you mind leaving a comment in the code to explain this too? Thanks, Justin!